### PR TITLE
Added definitions for "git-init" (supersedes #65055).

### PR DESCRIPTION
**Pull request by santi100a**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [X] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [X] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [X] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [X] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If adding a new definition:
- [X] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [X] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [X] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [X] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [X] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [X] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by santi100a** [comment]
Hey, @codediodeio! Can you please help me review this PR? @typescript-bot suggested me I should find someone to review it.

**Review by jakebailey — CHANGES_REQUESTED** [review]

File: types/git-init/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
+// Type definitions for git-init 1.0
+// Project: https://github.com/yoshuawuyts/git-init#readme
+// Definitions by: Santi <https://github.com/santi100a>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+interface Signals {
```
[the remark this answers is not in the document]
Comment (santi100a) [reply types/git-init/index.d.ts:5]: It won't let me import `node:child_process`, and installing `@types/node` has no effect.

[the remark this answers is not in the document]
Comment (santi100a) [reply types/git-init/index.d.ts:5]: Stay tuned -- if everything goes well, I'll push a new commit in a few hours.

**Review by jakebailey — APPROVED** [review]

**Comment by santi100a** [comment]
@typescript-bot Ready to merge! Go ahead, *MERGE IT*!

**Comment by santi100a** [comment]
@jakebailey, how long till `@types/git-init` is published?

**Comment by jakebailey** [comment]
Package publishes happen every half hour, so like 5 minutes

**Comment by santi100a** [comment]
Okay! Thank you, @jakebailey!

**Pull request merged**